src/controllers/capacity/index.ts

```diff
@@ -0,0 +1,93 @@
+import { Controller, Get, Post, Route, Path, Response, Body, SuccessResponse, Tags, Security } from 'tsoa'
+import { Logger } from 'pino'
+
+import { logger } from '../../lib/logger'
+import Database from '../../lib/db'
+import { DemandResponse, DemandSubtype, DemandRequest, DemandStatus } from '../../models/demand'
+import { UUID } from '../../models/uuid'
+import { NotFoundError } from '../../lib/error-handler/index'
```

Comment:
do we nee to suffix with Error, they are already in lib/error-handler (maybe errors would be more appropriate)

Reply:
removed them